tuna/flask_example.py

```diff
@@ -52,9 +51,11 @@ def get_table_example(grafana_req, data):
 
   #To populate the table with data from your DB:
   with DbSession() as session:
+    # pylint: disable=no-member ; @alex, conv_find_db has no rocm_v, miopen_v columns
```

Comment:
ConvolutionFindDB appears to not have any members 'rocm_v', 'miopen_v'. Could it be some foreign key join stuff going on...?

Reply:
these were also removed from the ConvolutionFindDB class, please remove any mentions of it in the code